src/js/serial.js

```diff
@@ -19,7 +19,8 @@ const serial = {
         const self = this;
         const testUrl = path.match(/^tcp:\/\/([A-Za-z0-9\.-]+)(?:\:(\d+))?$/);
         if (self.connectionId || self.connected) {
-            console.warn('We already connected. Aborting', self.connectionId, self.connected);
+            console.warn('We already connected. Aborting', self.connectionId, self.connected, new Error().stack);
```

Comment:
I don't know if the `new Error().stack` will be useful for something. What is the objective?

Reply:
To see where the connection was made from (but is only useful while debugging) - due to timeouts I could not step over the code fast enough, but I will remove it after getting more test results.